src/tsolvers/liasolver/Matrix.h

```diff
@@ -26,10 +26,10 @@ class LAVec
     opensmt::Real args[0]; // Either the elements or the relocation reference
 
 public:
-    LAVec(const vec<opensmt::Real>& ps, const opensmt::Real& den) : den(den) {
+    LAVec(std::vector<opensmt::Real>&& ps, const opensmt::Real& den) : den(den) {
         header.size = ps.size();
         header.reloced = 0;
-        for (int i = 0; i < ps.size(); i++) {
+        for (int i = 0; i < static_cast<int>(ps.size()); i++) {
```

Comment:
`int i` -> `std::size_t i` and there is no need for the cast.

Reply:
Fixed.